src/Project.WebMVC/Areas/Identity/Pages/Account/Manage/Index.cshtml.cs

```diff
@@ -0,0 +1,131 @@
+        }
+        
+        public async Task OnGetAsync(CancellationToken token)
+        {
+            var user = await _userManager.GetUserAsync(User);
+            
+            Input = new InputModel
+            {
+                Username = user.UserName,
+                PhoneNumber = user.PhoneNumber
+            };
+            
+            Languages = await GetEnabledLanguagesWithDefault(user.LanguageId, token);
+        }
+
+        public async Task OnPostAsync(CancellationToken token)
+        {
+            var user = await _userManager.GetUserAsync(User);
+            
+            if (Input.PhoneNumber != null && user.PhoneNumber != Input.PhoneNumber)
+            {
+                var phoneNumberResult =  await _userManager.SetPhoneNumberAsync(user, Input.PhoneNumber);
+                if (phoneNumberResult.Succeeded)
+                {
+                    _logger.LogInformation("Change phone number for user with id: {0}", user.Id);
+                }
+            }
+
+            if (user.LanguageId.ToString() != Input.LanguageId)
+            {
+                var language = await _context.Languages.FindAsync(Convert.ToInt32(Input.LanguageId));
+                
+                if (language != null)
+                {
+                    user.LanguageId = language.Id;
+                
+                    var changeLanguageResult = await _userManager.UpdateAsync(user);
+
+                    if (changeLanguageResult.Succeeded)
+                    {
+                        _logger.LogInformation("Change language for user with id: {0}", user.Id);
+                    }
+                
+                    var oldLanguageClaim = User.FindFirst("lang");
+                    var newLanguageClaim = new Claim("lang", language.Code);
+                    await _userManager.ReplaceClaimAsync(user, oldLanguageClaim, newLanguageClaim);
+                
+                    await _signInManager.RefreshSignInAsync(user);
+                }
+            }
+
+            Input = new InputModel
+            {   
+                Username = user.UserName,
+                PhoneNumber = user.PhoneNumber
+            };
+            
+            Languages = await GetEnabledLanguagesWithDefault(user.LanguageId, token);
+        }
+
+        [NonAction]
+        private async Task<IReadOnlyList<SelectListItem>> GetEnabledLanguagesWithDefault(
+            int defaultLanguageId, 
+            CancellationToken token = default)
+        {
+            var languages = await _context.Languages
+                .Where(language => language.Enabled)
+                .ToListAsync(token);
+
+            var userLanguage = languages.First(lang => lang.Id == defaultLanguageId);
+
+            var selectListOfLanguages = languages.Select(lang => new SelectListItem(lang.Name, lang.Id.ToString())).ToList();
+
+            var userLanguageSelectItem = selectListOfLanguages.First(item => item.Value == userLanguage.Id.ToString());
```

Comment:
You already iterating through all languages above, so you can set this flag as well, something like this:
`
var selectListOfLanguages = languages.Select(lang => new SelectListItem(lang.Name, lang.Id.ToString()) { Selected = lang.Id == userLanguage.Id}).ToList();
`